dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/workflow/WorkflowResourceIntegrationTest.java

```diff
@@ -657,4 +659,66 @@ public void testUpdateAction() throws Exception{
         assertEquals(actionNewName,wa.getName());
     }
 
+    /**
+     * Test the delete scheme resource
+     * @throws Exception
+     */
+    @SuppressWarnings("unchecked")
+    @Test
+    public void testDeleteScheme() throws Exception{
+        final Role adminRole = roleAPI.loadRoleByKey(ADMINISTRATOR);
+        final String adminRoleId = adminRole.getId();
+        final WorkflowScheme savedScheme = createScheme();
+        assertNotNull(savedScheme);
+        final List<WorkflowStep> workflowSteps = addSteps(savedScheme,2);
+        assertFalse(workflowSteps.isEmpty());
+        assertEquals(2, workflowSteps.size());
```

Comment:
In general let's try to keep the `asserts` to what the test is actually testing, for example these things like `addSteps` returning the correct number or not being empty are not inside the scope of the test (which is testing deleting a scheme), so they should be assumed working ok and not asserted, . This pollutes the testing with a high amount of asserts.